src/proc_fuse.c

```diff
@@ -164,17 +184,30 @@ __lxcfs_fuse_ops int proc_readdir(const char *path, void *buf,
 				  fuse_fill_dir_t filler, off_t offset,
 				  struct fuse_file_info *fi)
 {
-	if (dir_filler(filler, buf, ".",		0) != 0 ||
-	    dir_filler(filler, buf, "..",		0) != 0 ||
-	    dir_filler(filler, buf, "cpuinfo",		0) != 0 ||
-	    dir_filler(filler, buf, "meminfo",		0) != 0 ||
-	    dir_filler(filler, buf, "stat",		0) != 0 ||
-	    dir_filler(filler, buf, "uptime",		0) != 0 ||
-	    dir_filler(filler, buf, "diskstats",	0) != 0 ||
-	    dir_filler(filler, buf, "swaps",		0) != 0 ||
-	    dir_filler(filler, buf, "loadavg",		0) != 0 ||
-	    dir_filler(filler, buf, "slabinfo",		0) != 0)
-		return -EINVAL;
+	if (strcmp(path, "/proc") ==  0) {
+		if (dir_filler(filler, buf, ".",		0) != 0 ||
+		    dir_filler(filler, buf, "..",		0) != 0 ||
+		    dir_filler(filler, buf, "cpuinfo",		0) != 0 ||
+		    dir_filler(filler, buf, "meminfo",		0) != 0 ||
+		    dir_filler(filler, buf, "stat",		0) != 0 ||
+		    dir_filler(filler, buf, "uptime",		0) != 0 ||
+		    dir_filler(filler, buf, "diskstats",	0) != 0 ||
+		    dir_filler(filler, buf, "swaps",		0) != 0 ||
+		    dir_filler(filler, buf, "loadavg",		0) != 0 ||
+		    dir_filler(filler, buf, "slabinfo",		0) != 0 ||
+		    dirent_filler(filler, path, "pressure", buf, 0) != 0)
```

Comment:
I haven't tested this, but it looks like if the host does not have /proc/pressure, lxcfs will have a /proc/pressure though it will be empty.  I don't know whether that matters, especially as noone directly mounts /var/lib/lxcfs/proc into the container.